README.md

```diff
@@ -110,18 +110,18 @@ Where each location is on each platform:
 
 Worlds can be found as separate folders in: `user/worlds/`
 
-Configuration file:
--------------------
+Configuration file
+------------------
 - Default location:
     `user/minetest.conf`
-- It is created by Minetest when it is ran the first time.
+- It is created by Minetest when closing it for the first time.
```

Comment:
Still a bit weird. How about `This file is created by closing Minetest for the first time.`